### PR TITLE
Fix "whitespace" words - docs/standard

### PR DESCRIPTION
**Pull request by lucasnobredev**
## Fix "whitespace" words - docs/standard 

This PR came out from this [issue](https://github.com/dotnet/docs/issues/4439). I started with docs/standard because @wstaelens found an error on this section. 

## Brief explanation
We should use **white-space** for adjectives and **white space** for nouns.

## Suggested Reviewers

@mairaw @rpetrusha @wstaelens

### CODE REVIEW

File: docs/standard/base-types/regular-expression-options.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ ms.workload:
...
-|<xref:System.Text.RegularExpressions.RegexOptions.IgnorePatternWhitespace>|`x`|Exclude unescaped white space from the pattern, and enable comments after a number sign (`#`). For more information, see [Ignore Whitespace](#Whitespace).|  
+|<xref:System.Text.RegularExpressions.RegexOptions.IgnorePatternWhitespace>|`x`|Exclude unescaped white space from the pattern, and enable comments after a number sign (`#`). For more information, see [Ignore White Space](#Whitespace).|  
 |<xref:System.Text.RegularExpressions.RegexOptions.RightToLeft>|Not available|Change the search direction. Search moves from right to left instead of from left to right. For more information, see [Right-to-Left Mode](#RightToLeft).|  
 |<xref:System.Text.RegularExpressions.RegexOptions.ECMAScript>|Not available|Enable ECMAScript-compliant behavior for the expression. For more information, see [ECMAScript Matching Behavior](#ECMAScript).|  
 |<xref:System.Text.RegularExpressions.RegexOptions.CultureInvariant>|Not available|Ignore cultural differences in language. For more information, see [Comparison Using the Invariant Culture](#Invariant).|
```
Comment (rpetrusha) [inline docs/standard/base-types/regular-expression-options.md]: line 279 contains the text, "However, in the following cases, white space characters..." It should be, "However, in the following cases, white-space characters"

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you, @lucasnobredev, for contributing to the dotnet/docs repo and for beginning the work of correcting misuses of "white-space" and "white space". I recommended correcting occurrences of "whitespace". In reviewing the topic, however, I found two occurrences of "white space" that were misused. Would you mind correcting those as well?

File: docs/standard/base-types/character-classes-in-regular-expressions.md [hunk 2]
```diff
@@ -330,7 +330,7 @@ ms.workload:
 |\w+|Match one or more word characters.|
```
[the remark this answers is not in the document]
Comment (lucasnobredev) [reply docs/standard/base-types/character-classes-in-regular-expressions.md]: Done!

File: docs/standard/base-types/regular-expression-options.md (hunk 1 above)
Comment (lucasnobredev) [reply docs/standard/base-types/regular-expression-options.md]: Done! 2

**Review by lucasnobredev — COMMENTED** [review]
I focused on "whitespace" like you said. That's why I didn't see all those errors. Next time I'll pay more attention.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @lucasnobredev, and there's absolutely no need to apologize. This looks really good, and we appreciate the work you've put into it. We'll merge your PR now.

**Pull request merged**